main/src/library/Map.flix

```diff
@@ -752,13 +752,6 @@ namespace Map {
         let f = (acc, k, v) -> Map.insertWith(Set.union, v, Set#{k}, acc);
         Map.foldLeftWithKey(f, empty(), m)
 
-    ///
-    /// Returns `m` as a mutable set.
-    ///
-    @Time(1) @Space(1)
-    pub def toMutMap(m: Map[k, v]): MutMap[k, v] & Impure =
```

Comment:
I think it should be possible to keep this, if it looks like:

pub def toMutMap(m: Map[k, v]): MutMap[k, v, Impure] & Impure = .... as MutMap[k, v, Impure]

does that not work?

Reply:
Or something like this.

Reply:
I think if we restore `toMutMap` then we should almost be ready to merge this in.